Makefile

```diff
@@ -71,6 +71,11 @@ CRATE_MANIFESTS := $(addsuffix /Cargo.toml,$(CRATES))
 # Enumerate all .rs files in relevant crates
 RUST_RS_FILES := $(shell find $(CRATES) -type f -name '*.rs' 2>/dev/null | LC_ALL=C sort)
 
+# Files relevant to build interop
+INTEROP_RS_FILES := $(shell find interop -type f -name '*.rs' 2>/dev/null | LC_ALL=C sort)
```

Comment:
We also need `*.ts`, because we have those as real files that get parsed/built into interop at build time.